src/helpers/chainlink.js

```diff
@@ -0,0 +1,55 @@
+import { readJSONFile } from "./utils.js";
+import axios from "axios";
+import http from "http";
+
+const {
+  EI_CHAINLINKURL,
+  SUBMIT_RETRIES = 3,
+  CREDENTIALS_FILE = "config/ei_credentials.json",
+} = process.env;
+
+if (process.env.NODE_ENV !== "test") {
+  assert(EI_CHAINLINKURL, "$EI_CHAINLINKURL is required");
+  assert(CREDENTIALS_FILE !== "", '$CREDENTIALS_FILE is required');
+}
+
+/**
+ * Function to send a job run to the CL node
+ * @param {number} count the request id
+ * @param {string} jobId the Chainlink external job id
+ * @param {number} requestType the request type, 1 = time, 2 = deviation, 3 = *                             new round
```

Comment:
```suggestion
 * @param {number} requestType the request type, 1 = time, 2 = deviation, 3 = 
 *                             new round
```

Reply:
Fixed in https://github.com/jacquesvcritien/agoric-cl-middleware/commit/9f027c8cb34abf5790b78708b269ee97944c051e